yoti_python_sdk/config.py

```diff
@@ -23,3 +25,5 @@
 X_YOTI_SDK = "X-Yoti-SDK"
 X_YOTI_SDK_VERSION = X_YOTI_SDK + "-Version"
 JSON_CONTENT_TYPE = 'application/json'
+UTC_TIMEZONE = pytz.utc

```

Comment:
Will this only affect the tests?

Reply:
Yes, good point, it shouldn't be in the SDK if it's only used in tests. It's not actually used after the latest changes, so I'll remove it altogether.